backend/routes/events.py

```
@@ -0,0 +1,63 @@
+from fastapi import APIRouter, Depends, HTTPException, Request, Body, Query
```

Ruff (0.8.2)

`fastapi.HTTPException` imported but unused

Remove unused import

(F401)